### PR TITLE
iOS Deployment Target 11.0

### LINKED ISSUE #141: Build on Xcode 13
Now build failed on Xcode 13 ~~Beta~~, because could not use `@available` in Enum cases.

Drop iOS 9 and 10, then can remove `@available`.
I think if limiting to support iOS 11 and above will a very small affect us only.
The following data is iOS Usage by Apple:

https://developer.apple.com/support/app-store/

Screen Shot 2021-08-05 at 3 59 54

**Additional context**

### Environment
- Xcode: Version 13.0 beta 4 (13A233) / 13.0 (13A233)
- macOS: Big Sur 11.4 / 11.5.1

### Error Message

```
Enum cases with associated values cannot be marked potentially unavailable with '@available'
```

### Screenshots

Screen Shot 2021-08-03 at 4 56 55

### Developer Forums

[Enum cases with associated values cannot be marked potentially unavailable](https://developer.apple.com/forums/thread/682351)

### PR DESCRIPTION
**Pull request by ykws**
fixes #141

### CODE REVIEW

**Comment by ykws** [comment]
@el-hoshino Please check target OS on Bitrise, I guess remain to support iOS 9.

File: Tests/NotAutoLayoutTests/LayoutTest/ViewLayoutGuidesTest.swift [hunk 1]
```diff
@@ -128,7 +128,6 @@ class ViewLayoutGuidesTest: XCTestCase {
 		
 	}
 	
-	@available(iOS 11.0, *)
 	func testSafeAreaGuide() {
 		
 		// TODO: Find a way to add the view controller to the view hierarchy, otherwise the views cannot get the correct safe area settings.
```
Comment (el-hoshino-bitrise) [inline Tests/NotAutoLayoutTests/LayoutTest/ViewLayoutGuidesTest.swift:133]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/el-hoshino/NotAutoLayout/blob/e678db0f4b0c2d90555bc8723397c4f92ea63ce2/Tests/NotAutoLayoutTests/LayoutTest/ViewLayoutGuidesTest.swift#L133"></span>TODOs should be resolved (Find a way to add the view con…).<br />
<code>todo</code> <code>ViewLayoutGuidesTest.swift:133</code></td>
    </tr>
  </tbody>
</table>

**Comment by el-hoshino-bitrise** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/el-hoshino/NotAutoLayout/blob/e678db0f4b0c2d90555bc8723397c4f92ea63ce2/Sources/NotAutoLayout/Extensions/UIView.swift#L34">Sources/NotAutoLayout/Extensions/UIView.swift#L34</a> - FIXMEs should be resolved (Get this property without Auto…).<br />
<code>todo</code> <code>UIView.swift:34</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/el-hoshino/NotAutoLayout/blob/e678db0f4b0c2d90555bc8723397c4f92ea63ce2/Sources/NotAutoLayout/Extensions/UIView.swift#L87">Sources/NotAutoLayout/Extensions/UIView.swift#L87</a> - FIXMEs should be resolved (Get this property without Auto…).<br />
<code>todo</code> <code>UIView.swift:87</code></td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/el-hoshino/NotAutoLayout/blob/e678db0f4b0c2d90555bc8723397c4f92ea63ce2/Sources/NotAutoLayout/NotAutoLayoutCompatible/NotAutoLayoutCompatible.swift#L11">Sources/NotAutoLayout/NotAutoLayoutCompatible/NotAutoLayoutCompatible.swift#L11</a> - <strong><a href="https://github.com/ykws/NotAutoLayout/blob/e678db0f4b0c2d90555bc8723397c4f92ea63ce2/Sources/NotAutoLayout/NotAutoLayoutCompatible/NotAutoLayoutCompatible.swift#L11">Sources/NotAutoLayout/NotAutoLayoutCompatible/NotAutoLayoutCompatible.swift#L11</a></strong>: using 'class' keyword to define a class-constrained protocol is deprecated; use 'AnyObject' instead  <br /><code>
public protocol NotAutoLayoutCompatible: class {
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          2 Messages
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 26 tests, with 0 failures (0 unexpected) in 15.681 (15.709) seconds</td>
    </tr>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">What a good PR :white_flower:</td>
    </tr>
  </tbody>
</table>

## Develop PR Check Result

Item | Result
|--- | --- |
Head Branch check |:o:
Base Branch check |:o:
Merge Commits check |:o:
Modification Volumn check |:o:


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by el-hoshino** [comment]
@ykws Sorry, could you please close this PR and *remake* another PR to *develop* with the same head branch?
I tried to close and re-open this PR but somehow it just doesn't ignore the merge commits, which should already be included in develop 🤔

**Comment by ykws** [comment]
@el-hoshino Thank you for review. I force pushed remake PR from _develop_ branch, before from _master_ branch.

**Pull request merged**